### PR TITLE
Add script monitor alert

### PR DESCRIPTION
**Pull request by lumigar**
Adds the description for the script monitor new large script url alert

### CODE REVIEW

File: content/fundamentals/notifications/notification-available.md [hunk 1]
```diff
@@ -694,6 +694,30 @@ Investigate to confirm it is an expected change.
 </div>
 </details>
 
+<details>
+<summary>Script Monitor New Script Exceeds Max URL Length Alert</summary>
+<div>
+
+**Who is it for?**
+
+Page Shield customers who want to receive a notification when a script's URL exceeds the maximum allowed length
```
Comment (pedrosousa) [inline content/fundamentals/notifications/notification-available.md]:
```suggestion
Page Shield customers who want to receive a notification when a script's URL exceeds the maximum allowed length.
```

**Comment by marciocloudflare** [comment]
Hey @lumigar. What are you trying to add to the notification docs? Your PR changed since I first saw it, and the text regarding your description (new large script url alert) has vanished. Thanks!

**Comment by pedrosousa** [comment]
> Hey @lumigar. What are you trying to add to the notification docs? Your PR changed since I first saw it, and the text regarding your description (new large script url alert) has vanished. Thanks!

I'm checking this with both @lumigar and the PM, Márcio. Thank you for noticing!

**Comment by lumigar** [comment]
@pedrosousa @marciocloudflare sorry for the confusion, was expecting additional confirmation on this.  It's been confirmed now

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**